app/views/transactions/_btn_block.html.slim

```diff
@@ -0,0 +1,13 @@
+.col-sm-4.btn-group.pull-right
```

Comment:
@Gataniel it's not a good idea to use `col-sm` with `pull-right`. And move `col-sm` outside of partial.